rockcraft/models/project.py

```diff
@@ -326,6 +327,24 @@ def _validate_base_and_overlay(cls, item: Dict[str, Any], values) -> Dict[str, A
             )
         return item
 
+    @pydantic.validator("entrypoint_service", always=True)
+    @classmethod
+    def _validate_entrypoint_service(
+        cls, entrypoint_service: Optional[str], values: Any
+    ) -> Optional[str]:
+        """Verify that the entrypoint_service exist in the services dict."""
+        if entrypoint_service:
+            craft_cli.emit.message(
+                """Warning: defining an entrypoint-service will result in a rock with an atypical OCI Entrypoint. While that might be acceptable for testing and personal use, it shall require prior approval before submitting to a Canonical registry namespace."""
```

Comment:
how come the linters aren't complaining about the size of this line? :open_mouth:

Reply:
Maybe because it's a triple-quote string? Found a bypass 😎